test/boulder-config.json

```diff
@@ -127,7 +127,7 @@
     "maxContactsPerRegistration": 100,
     "debugAddr": "localhost:8002",
     "amqp": {
-      "server": "amqp://guest:guest@localhost:5673",
```

Comment:
Let's leave one of these as the old style so that it breaks if the code is broken in the future?